fe/fe-core/src/main/java/org/apache/doris/analysis/ShowProcStmt.java

```diff
@@ -46,14 +46,9 @@ public ProcNodeInterface getNode() {
     @Override
     public void analyze(Analyzer analyzer) throws AnalysisException {
         if (!Catalog.getCurrentCatalog().getAuth().checkGlobalPriv(ConnectContext.get(), PrivPredicate.ADMIN)) {
-            ErrorReport.reportAnalysisException(ErrorCode.ERR_SPECIFIC_ACCESS_DENIED_ERROR,
-                                                "ADMIN");
+            ErrorReport.reportAnalysisException(ErrorCode.ERR_SPECIFIC_ACCESS_DENIED_ERROR, "ADMIN");
         }
-
         node = ProcService.getInstance().open(path);
-        if (node == null) {
-            ErrorReport.reportAnalysisException(ErrorCode.ERR_WRONG_PROC_PATH, path);
-        }
     }
 
     @Override
```

Comment:
Could you replace code line 56-58 by the way, it includes redundant code.
```java
StringBuilder sb = new StringBuilder();
        sb.append("SHOW PROC '").append(path).append("'");
        return sb.toString();
```
with 

```java
        return "SHOW PROC '" + path + "'";
```

Reply:
@jackwener hi, I think both ways of coding can be ok. There is no clear specification when to use ‘+’ and when to use StringBuilder explicitly.